db/seeds.rb

```diff
@@ -7,4 +7,4 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 #
 # Remember to ensure that the commands in this file are idempotent!
-Instance.find_or_create_by(name: 'Default', host: '*')
+Instance.find_or_create_by(name: 'Default', host: 'coursemology.org')
```

Comment:
I changed default hostname for a reason that it is not valid, so it cannot be saved to db.. then tests failed..